spec/avram/queryable_spec.cr

```diff
@@ -62,6 +62,12 @@ describe Avram::Queryable do
     query.statement.should eq "SELECT #{User::COLUMN_SQL} FROM users WHERE users.age >= $1 AND users.name = $2"
   end
 
+  it "releases connection if no open transaction", tags: Avram::SpecHelper::TRUNCATE do
```

Comment:
Exactly like https://github.com/luckyframework/avram/pull/800 we use truncation to avoid having a transaction on the connection by default